### ISSUE TITLE
Pretrained-Models Release

### ISSUE THREAD

**Issue by tushaarkataria** [MLNeurosurg/hidisc#1]
Hello,

Are you going to release pre-trained models with different contrastive techniques ?? Or even just the ones obtained using the proposed method??

**Comment by Hasindri** [comment]
Yes, could you please make the pretrained weights for ResNet50 model on SRH data available? Thank you.
